apps/full-site-editing/full-site-editing-plugin/premium-content/blocks/login-button/color-edit.js

```diff
@@ -0,0 +1,213 @@
+// The code in this file is copied entirely from https://github.com/WordPress/gutenberg/blob/master/packages/block-library/src/button/color-edit.js
```

Comment:
Code copied from Gutenberg, feel free to skip during review (it'd be possible to get rid of this once the Core Button block can use the new colors hook: https://github.com/WordPress/gutenberg/issues/22700).